dags/dag_functions.py

```diff
@@ -129,3 +129,10 @@ def task_fail_slack_alert(
         proxy=proxy,
     )
     return failed_alert.execute(context=context)
+
+def get_readme_docmd(readme_path, dag_name):
+    import re
+    contents = open(readme_path, 'r').read()
+    doc_md_key = '<!-- ' + dag_name + '_doc_md -->'
```

Comment:
Can you update the [Contributing.md](https://github.com/CityofToronto/bdit_data-sources/blob/master/Contributing.md) file to instruct all of us to do this while writing READMEs for DAGs

Reply:
Added to #806